src/commands/apps/channels/create.ts

```diff
@@ -63,9 +67,13 @@ export default defineCommand({
       consola.success('Channel created successfully.');
       consola.info(`Channel ID: ${response.id}`);
     } catch (error) {
-      const message = getMessageFromUnknownError(error);
-      consola.error(message);
-      process.exit(1);
```

Comment:
Please keep the error message and just return error code 0 if the param was passed.